### ISSUE TITLE
[BUG] API REST return 500 and not response of Axios

### ISSUE THREAD

**Issue by Paul75** [TypedProject/tsed#1377]
Hello,

I do this : 
```
const url = config.lescrous.url + '/tapioca/v1/' + params.url_service + '/' + params.crous_id + '/' + params.ug_id + '/' + params.ud_id + '/' + params.date_id + '/' + params.service_id;

        return Axios.get(url, {
            responseType: "json",
            headers: {
                'Content-Type': 'application/x-www-form-urlencoded',
                'Authorization': token,
            }
        });

```
And it return error 500 not the response of Axios ...

Thanks.

**Comment by Romakita** [comment]
Hello @Paul75 

The doc describe how the axios can be used with Ts.ED as a proxy:
https://tsed.io/docs/controllers.html#axios-response

```typescript
   return Axios.get(`https://cerevoice.s3.amazonaws.com/${path}`, {
      responseType: "stream"
    });
```
You've missed the responseType: `stream`. There is agreement from the doc to support the Json responseType: "json".

See you
Romain

**Comment by Paul75** [comment]
Hello,

But I do like doc : 
```
const url = config.lescrous.url + '/tapioca/v1/' + params.url_service + '/' + params.crous_id + '/' + params.ug_id + '/' + params.ud_id + '/' + params.date_id + '/' + params.service_id;

        return Axios.get(url, {
            responseType: "stream",
            headers: {
                'Content-Type': 'application/x-www-form-urlencoded',
                'Authorization': token,
            }
        });
```

And got  : 
`{"name":"Error","message":"Request failed with status code 403","status":500,"errors":[],"stack":"Error: Request failed with status code 403\n    at createError (/home/pkbp512/www/lucy/garone_menus/gm_server/node_modules/axios/lib/core/createError.js:16:15)\n    at settle (/home/pkbp512/www/lucy/garone_menus/gm_server/node_modules/axios/lib/core/settle.js:17:12)\n    at RedirectableRequest.handleResponse (/home/pkbp512/www/lucy/garone_menus/gm_server/node_modules/axios/lib/adapters/http.js:231:9)\n    at RedirectableRequest.emit (events.js:315:20)\n    at RedirectableRequest._processResponse (/home/pkbp512/www/lucy/garone_menus/gm_server/node_modules/follow-redirects/index.js:403:10)\n    at ClientRequest.RedirectableRequest._onNativeResponse (/home/pkbp512/www/lucy/garone_menus/gm_server/node_modules/follow-redirects/index.js:57:10)\n    at Object.onceWrapper (events.js:422:26)\n    at ClientRequest.emit (events.js:315:20)\n    at HTTPParser.parserOnIncomingClient (_http_client.js:641:27)\n    at HTTPParser.parserOnHeadersComplete (_http_common.js:126:17)\n    at TLSSocket.socketOnData (_http_client.js:509:22)\n    at TLSSocket.emit (events.js:315:20)\n    at addChunk (internal/streams/readable.js:309:12)\n    at readableAddChunk (internal/streams/readable.js:284:9)\n    at TLSSocket.Readable.push (internal/streams/readable.js:223:10)\n    at TLSWrap.onStreamRead (internal/stream_base_commons.js:188:23)"}`

Thanks

**Comment by Romakita** [comment]
Ok now your issue more clear guys :)
With more details I can investigate correctly!!!

**Comment by Romakita** [comment]
@Paul75  Prochaine fois, fais moi une vraie issue github avec des détails sur la réponse back que tu as, ce que tu attends et un cas reproductible serait encore mieux ;) (même si tu n'as pas le temps, car je ne suis pas payé non plus pour aider).

Je regarde ton soucis
Romain

**Comment by Romakita** [comment]
PR: https://github.com/TypedProject/tsed/pull/1378

**Comment by Romakita** [comment]
responseType: json is now supported correctly.

**Comment by Romakita** [comment]
Release in progress :)

**Comment by Paul75** [comment]
Bonjour 

Merci cela fonctionne 

4 mai 2021 17:46:04 Romain Lenzotti ***@***.***>:

> 🎉 This issue has been resolved in version 6.47.2 🎉
> 
> The release is available on:
> 
* > *v6.47.2*
* > GitHub release[https://github.com/TypedProject/tsed/releases/tag/v6.47.2]
